### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/js/components/Article.js [hunk 1]
```diff
@@ -28,4 +30,4 @@ export default class Article extends Component {
       </div>
     )
   }
-}
\ No newline at end of file
+}
```
Comment (skliarov) [inline src/js/components/Article.js:33]: Yura, configure your editor to NOT automatically add new lines at the end of the document. Is it hard to just copy config from gist? It's third time I'm asking :(

File: src/js/components/ArticleForm.js [hunk 2]
```diff
@@ -96,4 +96,4 @@ export default class ArticleForm extends Component {
       </div>
     )
   }
-}
\ No newline at end of file
+}
```
Comment (skliarov) [inline src/js/components/ArticleForm.js:99]: Yura, configure your editor to NOT automatically add new lines at the end of the document. Is it hard to just copy config from gist? It's third time I'm asking :(

File: src/js/components/Articles.js [hunk 3]
```diff
@@ -51,4 +51,4 @@ export default class Articles extends Component {
       </div>
     )
   }
-}
\ No newline at end of file
+}
```
Comment (skliarov) [inline src/js/components/Articles.js:54]: Yura, configure your editor to NOT automatically add new lines at the end of the document. Is it hard to just copy config from gist? It's third time I'm asking :(

File: src/js/components/EditArticle.js [hunk 4]
```diff
@@ -28,4 +30,4 @@ export default class EditArticle extends React.Component {
       <ArticleForm handleSubmit={ this.handleSubmit.bind(this) } ref='articleForm' />
     )
   }
-}
\ No newline at end of file
+}
```
Comment (skliarov) [inline src/js/components/EditArticle.js:33]: Yura, configure your editor to NOT automatically add new lines at the end of the document. Is it hard to just copy config from gist? It's third time I'm asking :(

File: src/js/components/NewArticle.js [hunk 5]
```diff
@@ -20,4 +20,4 @@ export default class NewArticle extends Component {
       <ArticleForm handleSubmit={ this.handleSubmit.bind(this) } />
     )
   }
-}
\ No newline at end of file
+}
```
Comment (skliarov) [inline src/js/components/NewArticle.js:23]: Yura, configure your editor to NOT automatically add new lines at the end of the document. Is it hard to just copy config from gist? It's third time I'm asking :(

File: src/js/stores/AppState.js [hunk 6]
```diff
@@ -6,19 +6,20 @@ const API_URL = 'https://serene-ocean-15499.herokuapp.com/api/v1'
 export default class AppState {
   @observable articles = []
   @observable article = {}
-  
+
```
Comment (skliarov) [inline src/js/stores/AppState.js:9]: Don't remove spaces from empty lines, third time as well :(

File: src/js/stores/AppState.js [hunk 7]
```diff
@@ -57,4 +58,4 @@ export default class AppState {
 
     return request
   }
-}
\ No newline at end of file
+}
```
Comment (skliarov) [inline src/js/stores/AppState.js:61]: Yura, configure your editor to NOT automatically add new lines at the end of the document. Is it hard to just copy config from gist? It's third time I'm asking :(

File: src/js/stores/AppState.js [hunk 8]
```diff
@@ -6,19 +6,20 @@ const API_URL = 'https://serene-ocean-15499.herokuapp.com/api/v1'
...
     request.then((response) => {
-      // TODO: Handle errors here
-      this.articles = response.data
+      if (response.status == 200) {
+        this.articles = response.data
+      }
     })
   }
-  
+
```
Comment (skliarov) [inline src/js/stores/AppState.js:22]: Don't remove spaces from empty lines, third time as well :(

File: src/js/stores/AppState.js [hunk 9]
```diff
@@ -27,7 +28,7 @@ export default class AppState {
     })
     return request
   }
-  
+
```
Comment (skliarov) [inline src/js/stores/AppState.js:31]: Don't remove spaces from empty lines, third time as well :(

File: src/js/stores/AppState.js [hunk 10]
```diff
@@ -37,7 +38,7 @@ export default class AppState {
       this.loadArticles()
     })
   }
-  
+
```
Comment (skliarov) [inline src/js/stores/AppState.js:41]: Don't remove spaces from empty lines, third time as well :(

File: src/js/stores/AppState.js [hunk 11]
```diff
@@ -47,7 +48,7 @@ export default class AppState {
 
     return request
   }
-  
+
```
Comment (skliarov) [inline src/js/stores/AppState.js:51]: Don't remove spaces from empty lines, third time as well :(

File: src/js/components/EditArticle.js [hunk 12]
```diff
@@ -11,10 +11,12 @@ export default class EditArticle extends React.Component {
...
     let articleForm = this.refs.articleForm
     this.props.appState.fetchArticle(articleID).then((response) => {
-      articleForm.setArticle(response.data)
+      if (response.status == 200) {
+        articleForm.setArticle(response.data)
+      }
     })
   }
-  
+
```
Comment (skliarov) [inline src/js/components/EditArticle.js:19]: Don't remove spaces from empty lines, third time as well :(

File: src/js/components/Articles.js [hunk 13]
```diff
@@ -30,7 +30,7 @@ export default class Articles extends Component {
       )
     })
   }
-
+
```
Comment (skliarov) [inline src/js/components/Articles.js]: Don't remove spaces from empty lines, third time as well :(

File: src/js/components/ArticleForm.js [hunk 14]
```diff
@@ -41,7 +41,7 @@ export default class ArticleForm extends Component {
   
   handleSubmit(event) {
     event.preventDefault()
-    
+
```
Comment (skliarov) [inline src/js/components/ArticleForm.js:44]: Don't remove spaces from empty lines, third time as well :(

**Comment by ghost** [comment]
it using your config... i coppied it... i dont know why its adding... strange...

**Pull request merged**